### PR TITLE
Add deleteEvent command

### PR DESCRIPTION
**Pull request by vishandi**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/schedule/DeleteEventCommand.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        }
+
+        Person personToEdit = lastShownList.get(targetIndex.getZeroBased());
+        Schedule scheduleToEdit = personToEdit.getSchedule();
+
+        if (targetEventIndex.getZeroBased() >= scheduleToEdit.getEvents().size()) {
+            throw new CommandException(MESSAGE_INVALID_EVENT_DISPLAYED_INDEX);
```
Comment (hanqinilnix) [inline src/main/java/seedu/address/logic/commands/schedule/DeleteEventCommand.java:59]:
```suggestion
            throw new CommandException(Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX);
```
Try to keep consistency with the previous CommandException on line 52

Comment (hanqinilnix) [reply src/main/java/seedu/address/logic/commands/schedule/DeleteEventCommand.java:59]: We can either change this or the one on line 52

Comment (hanqinilnix) [reply src/main/java/seedu/address/logic/commands/schedule/DeleteEventCommand.java:59]: The same should be done in EditEventCommand

**Review by hanqinilnix — COMMENTED** [review]
Good job! No jarring problems that can be found from reading. Even included test classes, nice!

File: src/main/java/seedu/address/logic/commands/schedule/DeleteEventCommand.java (hunk 1 above)
Comment (vishandi) [reply src/main/java/seedu/address/logic/commands/schedule/DeleteEventCommand.java:59]: Line 59 changed on the next commit.

**Pull request merged**